include/c_common/get_check_data.h

```diff
@@ -30,107 +30,107 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 typedef struct Column_info_t Column_info_t;
 
 /*!
-@brief  Function will check whether the colNumber represent any specific column or NULL (SPI_ERROR_NOATTRIBUTE).
-
-@param[in] colNumber Column number (count starts at 1).
-
-@return @b TRUE when colNumber exist.
-        @b FALSE when colNumber was not found.
-
+ * @brief  Function will check whether the colNumber represent any specific column or NULL (SPI_ERROR_NOATTRIBUTE).
+ *
+ * @param[in] colNumber Column number (count starts at 1).
+ *
+ * @return @b TRUE when colNumber exist.
+ *        @b FALSE when colNumber was not found.
+ *
  */
 bool column_found(int colNumber);
 
 /*!
-@brief Function tells expected type of each column and then check the correspondence type of each column.
-
-@param[in] info[]     contain one or more column information.
-@param[in] info_size  number of columns.
-
-@throw ERROR Unknown type of column.
-
-@return NULL is always returned.
-
+ * @brief Function tells expected type of each column and then check the correspondence type of each column.
+ *
+ * @param[in] info[]     contain one or more column information.
+ * @param[in] info_size  number of columns.
+ * 
+ * @throw ERROR Unknown type of column.
+ *
+ * @return NULL is always returned.
+ *
  */
 void pgr_fetch_column_info(
         Column_info_t info[],
         int info_size);
 
 /*!
-@brief The function check whether column type is ANY-INTEGER or not.
-       Where ANY-INTEGER is SQL type:
-            SMALLINT, INTEGER, BIGINT
-
-@param[in] info contain column information.
-
-@throw ERROR Unexpected Column type. Expected column type is ANY-INTEGER.
-
+ * @brief The function check whether column type is ANY-INTEGER or not.
+ *       Where ANY-INTEGER is SQL type:
+ *           SMALLINT, INTEGER, BIGINT
+ * 
```

Comment:
This line has a trailing space:
There should be no space at the end of the line